Remove test_large_set.py from the tests folder

Currently, a `test_large_set.py` file is present in the test folder, making the tests fail if the large test data is not present. This prevents to run the tests with a simple command like `poetry run pytest` as we're used to do with python programs.

The large test suite is really a great tool, but it's impractical to run on a day to day basis, removing it from this folder and having another way to run it (cf #969) seems a better way to handle these.

```traceback
=========================================================================== ERRORS ===========================================================================
__________________________________________________________ ERROR collecting tests/test_large_set.py __________________________________________________________
tests/test_large_set.py:42: in <module>
    docs_10K = get_test_docs(min_size=0, max_size=10 * 2**10)
tests/test_large_set.py:29: in get_test_docs
    ensure_test_data_exists()
tests/test_large_set.py:25: in ensure_test_data_exists
    sys.exit(1)
E   SystemExit: 1
---------------------------------------------------------------------- Captured stdout -----------------------------------------------------------------------
Test data repository it empty. Skipping large tests.
```

--- a/.gitignore
+++ b/.gitignore
@@ -145,7 +145,6 @@ debian/dangerzone.*
 deb_dist
 .DS_Store
 tests/test_docs/**/*-safe.pdf
-tests/test_docs_large/
 install/windows/Dangerzone.wxs
 share/container.tar
 share/container.tar.gz

--- a/.gitmodules
+++ b/.gitmodules
@@ -1,3 +1,0 @@
-[submodule "tests/test_docs_large"]
-	path = tests/test_docs_large
-	url = https://github.com/freedomofpress/dangerzone-test-set

--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,3 @@
-LARGE_TEST_REPO_DIR:=tests/test_docs_large
-GIT_DESC=$$(git describe)
-JUNIT_FLAGS := --capture=sys -o junit_logging=all
-
 .PHONY: lint
 lint: ## Check the code for linting, formatting, and typing issues with ruff and mypy
 	ruff check
@@ -19,25 +15,7 @@ test: ## Run the tests
 	# shared state.
 	# See more in https://github.com/freedomofpress/dangerzone/issues/493
 	pytest --co -q tests/gui | grep -e '^tests/' | xargs -n 1 pytest -v
-	pytest -v --cov --ignore dev_scripts --ignore tests/gui --ignore tests/test_large_set.py
-
-
-.PHONY: test-large-requirements
-test-large-requirements:
-	@git-lfs --version || (echo "ERROR: you need to install 'git-lfs'" && false)
-	@xmllint --version || (echo "ERROR: you need to install 'xmllint'" && false)
-
-test-large-init: test-large-requirements
-	@echo "initializing 'test_docs_large' submodule"
-	git submodule init $(LARGE_TEST_REPO_DIR)
-	git submodule update $(LARGE_TEST_REPO_DIR)
-	cd $(LARGE_TEST_REPO_DIR) && $(MAKE) clone-docs
-
-TEST_LARGE_RESULTS:=$(LARGE_TEST_REPO_DIR)/results/junit/commit_$(GIT_DESC).junit.xml
-.PHONY: test-large
-test-large: test-large-init  ## Run large test set
-	python -m pytest --tb=no tests/test_large_set.py::TestLargeSet -v $(JUNIT_FLAGS) --junitxml=$(TEST_LARGE_RESULTS)
-	python $(TEST_LARGE_RESULTS)/report.py $(TEST_LARGE_RESULTS)
+	pytest -v --cov --ignore dev_scripts --ignore tests/gui
 
 Dockerfile: Dockerfile.env Dockerfile.in ## Regenerate the Dockerfile from its template
 	poetry run jinja2 Dockerfile.in Dockerfile.env > Dockerfile

--- a/dev_scripts/qa.py
+++ b/dev_scripts/qa.py
@@ -818,9 +818,7 @@ class QAWindows(QABase):
     @QABase.task("Run tests", ref="REF_BUILD", auto=True)
     def run_tests(self):
         # NOTE: Windows does not have Makefile by default.
-        self.run(
-            "poetry", "run", "pytest", "-v", "--ignore", r"tests\test_large_set.py"
-        )
+        self.run("poetry", "run", "pytest", "-v")
 
     @QABase.task("Build Dangerzone .exe", ref="REF_BUILD", auto=True)
     def build_dangerzone_exe(self):

--- a/docs/developer/TESTING.md
+++ b/docs/developer/TESTING.md
@@ -11,28 +11,3 @@ Unit / integration tests are run with:
 ```bash
 poetry run make test
 ```
-
-## Run large tests
-
-We also have a larger set of tests that can take a day or more to run, where we evaluate the completeness of Dangerzone conversions.
-
-```bash
-poetry run make test-large
-```
-
-### Test report generation
-After running the large tests, a report is stored under `tests/test_docs_large/results/junit/` and it is composed of the JUnit XML file describing the pytest run.
-
-This report can be analysed for errors. It is obtained by running:
-
-```bash
-cd tests/docs_test_large
-make report
-```
-
-If you want to run the report on some historical test result, you can call:
-
-```bash
-cd tests/docs_test_large
-python report.py tests/test_docs_large/results/junit/commit_<COMMIT_ID>.junit.xml
-```

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -95,7 +95,6 @@ exclude = [
 	"dangerzone/podman/command/",
 	"dangerzone/podman/errors/",
 	"dangerzone/podman/errors/",
-	"tests/test_docs_large/",
 ]
 
 [tool.ruff.lint]

